### PR TITLE
SR fashion update featuring Bing Guss

### PR DESCRIPTION
**Pull request by MagnusCrowe**
## About the PR

**New green clothes for station representative made by Bing Guss:**
- Jumpsuit
- Jumpskirt
- Cloak
- Mantle
- Hat
- Jacket
- Carapace
- PDA

**New green clothes for station guard made by MagnusCrowe:**
- Jumpsuit
- Jumpskirt

**New blue clothes for private security made by MagnusCrowe:**
- Jumpsuit
- Jumpskirt

**Other changes:**
- Added a dresser filled with spare clothes for the SR and SG
- Reverted HoP clothing to be HoP clothing again
- Added a new directional sign for SR office

## Why / Balance

We thought the SR had drip, turns out it was swag!
This issue is resolved with this PR.

## Media

image
image
image
image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: MagnusCrowe
- add: Added new uniform for station guard and private security.

:cl: GentleButter
- add: Added a full suite of clothing for the station representative.

### CODE REVIEW

**Comment by dvir001** [comment]
Upstream removing MONK clothes, need to see if the PR approved..

**Comment by dvir001** [comment]
@MagnusCrowe Can you just yeet SR into a new job file, have HOP stay as upstream with a change to make it "NoSpawn" so people cannot pick it and replace the job slot on the frontier.

We did it with sheriff now.

**Comment by MagnusCrowe** [comment]
> @MagnusCrowe Can you just yeet SR into a new job file, have HOP stay as upstream with a change to make it "setPreference: false" so people cannot pick it and replace the job slot on the frontier.
> 
> We did it with sheriff now.
> 
> TLDR, restore `head_of_personnel.yml` to upstream state. Replace on the time tracker line to `playTimeTracker: JobHeadOfPersonnelOld # Frontier` and add this under _NF Add `setPreference: false # Frontier` Add new file named sr.yml so we can upkeep this without conflicts. Job ID is now actually SR.
> 
> If you dont have time for this ill try to do this myself later today.

If you want to, go ahead. I should probably roll the station update into this PR too huh?

I need to replace the signs and recolor the SR office.

**Comment by dvir001** [comment]
Do another pr right after, less hassle.

**Review by dvir001 — APPROVED** [review]
Editing this game me actual pain

Good to go
